contracts/src/v0.8/dev/automation/upkeeps/MercuryRegistryBatchUpkeep.sol

```diff
@@ -0,0 +1,80 @@
+pragma solidity 0.8.6;
+
+import "../../../shared/access/ConfirmedOwner.sol";
+import "../../../automation/interfaces/AutomationCompatibleInterface.sol";
+import "../2_1/interfaces/FeedLookupCompatibleInterface.sol";
+import "./MercuryRegistry.sol";
+
+contract MercuryRegistryBatchUpkeep is ConfirmedOwner, AutomationCompatibleInterface, FeedLookupCompatibleInterface {
+  error BatchSizeTooLarge(uint256 batchsize, uint256 maxBatchSize);
+  // Use a reasonable maximum batch size. Every Mercury report is ~750 bytes, too many reports
+  // passed into a single batch could exceed the calldata or transaction size limit for some blockchains.
+  uint256 constant MAX_BATCH_SIZE = 50;
+
+  MercuryRegistry immutable i_registry; // master registry, where feed data is stored
+
+  uint256 s_batchStart; // starting index of upkeep batch, inclusive
+  uint256 s_batchEnd; // ending index of upkeep batch, exclusive
```

Comment:
These seem like indexes into the `s_feeds` array in the MercuryRegistry contract - maybe can state that more clearly in the comment here?

Reply:
will do.